app/src/main/java/com/moviefiend/torian/moviefiend/MainActivity.java

```diff
@@ -1,13 +1,26 @@
 package com.moviefiend.torian.moviefiend;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
-public class MainActivity extends AppCompatActivity {
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+public class MainActivity extends AppCompatActivity implements MovieListFragment.Listener {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        MovieListFragment fragment = (MovieListFragment) getFragmentManager().findFragmentById(R.id.movie_list_fragment);
+        fragment.setListener(this);
```

Comment:
No need for this method, in the fragment you can simply call `getActivity`.
More than that it can be somewhat dangerous to have a `setListener` method as other developers can use it from places other than (`onCreate`) and not be aware that once the activity gets destructed then the fragment will have a stale reference.